tests/unit/amo/components/TestHomeHero.js

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+
+import HomeHero, { HomeHeroBase } from 'amo/components/HomeHero';
+import {
+  dispatchClientMetadata,
+  fakeI18n,
+  shallowUntilTarget,
+} from 'tests/unit/helpers';
+import Hero from 'ui/components/Hero';
+
+describe(__filename, () => {
+  const defaultProps = {
+    i18n: fakeI18n(),
+    store: dispatchClientMetadata().store,
+  };
+
+  function render(props) {
+    return shallowUntilTarget(
+      <HomeHero {...defaultProps} {...props} />,
+      HomeHeroBase,
+    );
+  }
+
+  it('renders a HomeHero', () => {
+    const root = render();
+    expect(root).toHaveClassName('HomeHero');
```

Comment:
You can add some more assertions for the title, subtitle, etc. No need to do exact matches, it should just be there.